src/common/fetch.ts

```diff
@@ -1,22 +0,0 @@
-/**
- * Copyright 2020 Google Inc. All rights reserved.
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-import { isNode } from '../environment.js';
-
-/* Use the global version if we're in the browser, else load the node-fetch module. */
-export const getFetch = async (): Promise<typeof fetch> => {
```

Comment:
let's keep conditional imports here as in DevTools (browser version) we cannot have any dependencies as we don't use node module resolution.